bot/cogs/trivia.py

```diff
@@ -19,6 +22,15 @@ class TriviaCog(commands.Cog):
 
     def __init__(self, bot):
         self.bot = bot
+        global scores
```

Comment:
do not use globals. this should be a class var if it needs to be accessed globally. i.e 
```
class TriviaCog(commands.Cog):
    """
    Commands for the Trivia Questions
    """
    scores = {}
```